### ISSUE TITLE
Django 3.0 support

### ISSUE THREAD

**Issue by MattFisher** [pinax/django-user-accounts#321]
When running our test suite with `django-user-accounts` v2.1.0 and Django 3.0, the line `from django.utils import six, timezone, translation` when importing `account.models` results in `ImportError: cannot import name 'six'`

```
Traceback (most recent call last):
--
  | File "./manage.py", line 15, in <module>
  | execute_from_command_line(sys.argv)
  | File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
  | utility.execute()
  | File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 377, in execute
  | django.setup()
  | File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
  | apps.populate(settings.INSTALLED_APPS)
  | File "/usr/local/lib/python3.6/site-packages/django/apps/registry.py", line 114, in populate
  | app_config.import_models()
  | File "/usr/local/lib/python3.6/site-packages/django/apps/config.py", line 211, in import_models
  | self.models_module = import_module(models_module_name)
  | File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
  | return _bootstrap._gcd_import(name[level:], package, level)
  | File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  | File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  | File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  | File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  | File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  | File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  | File "/[myapp]/models.py", line 76, in <module>
  | from [myapp].tasks import [thing]
  | File "/[myapp]/tasks.py", line 27, in <module>
  | from [myapp].services import [other_thing]
  | File "/[myapp]/services/__init__.py", line 5, in <module>
  | from [myapp].tool import (
  | File "[myapp]/services/tool.py", line 6, in <module>
  | from account.models import EmailAddress
  | File "/usr/local/lib/python3.6/site-packages/account/models.py", line 12, in <module>
  | from django.utils import six, timezone, translation
  | ImportError: cannot import name 'six'
```

I'd be happy to raise a PR if that would be helpful.

**Comment by MattFisher** [comment]
Cherry-picking [this commit](https://github.com/massware/django-user-accounts/commit/0d9d407b9bd5414f1c5a03b9e0cfdd86407f1a2b) should get pretty close.

**Comment by delzac** [comment]
Hi, can i check if there's any plans/timeline to support django 3.0?

**Comment by KatherineMichel** [comment]
@MattFisher @delzac We've updated the supported versions, including Django 3.0. It has been published to PyPi as DUA 3.0. Let us know if you have any problems. Closing issue. https://pypi.org/project/django-user-accounts/#history
